### PR TITLE
Fix Codeowners file.

### PR DESCRIPTION
**Pull request by IMB11**
Allows people to edit sidebar without needing developer review.

### CODE REVIEW

**Review by modmuss50 — CHANGES_REQUESTED** [review]
Can you just add the sidebar file/folders instead of defining everything else.

**Comment by IMB11** [comment]
~~You can't, you'd still require the review from Developers as they have the whole .vitepress folder - also, the `!` negate syntax doesn't work with codeowners files.~~

Turns out exclusion syntax is a bit odd, but it's there.

**Pull request merged**
